test/utils.jl

```diff
@@ -80,34 +77,25 @@ module TestUtils
             3rd Quartile:   3.250000
             Maximum:        4.000000
             Length:         4
-            Type:           $Int
+            Type:           Int64
 
             nullarr
             Summary Stats:
-            Mean:           3.500000
-            Minimum:        2.000000
-            1st Quartile:   2.750000
-            Median:         3.500000
-            3rd Quartile:   4.250000
-            Maximum:        5.000000
             Length:         4
-            Type:           $Int
-            Number Missing: 0
-            % Missing:      0.000000
+            Type:           Union{Int64, Nulls.Null}
```

Comment:
`Nulls.jl` is pretty small, so that might be ok; we could also provide our own `describe` for `?Real`. In fact, I think we might need to basically copy the `NullableArray` definitions and either put them in DataTables or Nulls.jl itself for full parity. @nalimilan thoughts?